tutorials/joint_controller.md

```diff
@@ -0,0 +1,685 @@
+              <size>0.5 0.5 0.01</size>
+            </box>
+          </geometry>
+        </visual>
+        <collision name="base_collision">
+          <pose>0.0 0.0 0.0 0 0 0</pose>
+          <geometry>
+            <box>
+              <size>0.5 0.5 0.01</size>
+            </box>
+          </geometry>
+        </collision>
+      </link>
+      <link name="rotor">
+        <pose>0.0 0.0 0.1 0 0 0</pose>
+        <inertial>
+          <inertia>
+            <ixx>0.032</ixx>
+            <ixy>0</ixy>
+            <ixz>0</ixz>
+            <iyy>0.032</iyy>
+            <iyz>0</iyz>
+            <izz>0.00012</izz>
+          </inertia>
+          <mass>0.6</mass>
+        </inertial>
+        <visual name="visual">
+          <geometry>
+            <box>
+              <size>0.25 0.1 0.05</size>
+            </box>
+          </geometry>
+          <material>
+            <ambient>0.2 0.8 0.2 1</ambient>
+            <diffuse>0.8 0 0 1</diffuse>
+          </material>
+        </visual>
+        <collision name="collision">
+          <geometry>
+            <box>
+              <size>0.25 0.1 0.05</size>
+            </box>
+          </geometry>
+        </collision>
+      </link>
+
+      <joint name="world_fixed" type="fixed">
+        <parent>world</parent>
+        <child>base_link</child>
+      </joint>
+
+      <joint name="j1" type="revolute">
+        <pose>0 0 -0.5 0 0 0</pose>
+        <parent>base_link</parent>
+        <child>rotor</child>
+        <axis>
+          <xyz>0 0 1</xyz>
+        </axis>
+      </joint>
+</model>
+</world>
+</sdf>
+```
+
+3) Run the following command to launch the gazebo simulation:
+
+```bash
+gz sim -v 4 -r example.sdf
+```
+
+This is how the model will look:
+
+<div style="text-align:center;">
+  \image html files/joint_controllers/JointController.png width=50%
+</div>
+
+4) Now let's add the Gazebo JointController plugin to the SDF file. Add the following line to your file just before the tag `</model>`.
+
+- Velocity mode
+
+```xml
+<plugin
+ filename="gz-sim-joint-controller-system"
+ name="gz::sim::systems::JointController">
+ <joint_name>j1</joint_name>
+ <initial_velocity>1.0</initial_velocity>
+</plugin>
+```
+
+The initial velocity is set to 1.0 rad/s.
+
+<div style="text-align:center;">
+  \image html files/joint_controllers/JointControllerVelMode1.gif width=50%
+</div>
+
+One can change the joint velocity by publishing a message on the topic `/model/joint_controller_demo/joint/j1/cmd_vel` or can change the topic name within the plugin.
+
+To change the topic name add following line before ```</plugin>``` tag in SDF file.
+
+```xml
+<topic>topic_name</topic>
+```
+
+- Sending velocity commands
+
+```bash
+gz topic -t "/topic_name" -m gz.msgs.Double -p "data: 10.0"
+```
+
+<div style="text-align:center;">
+  \image html files/joint_controllers/JointControllerVelMode2.gif width=50%
+</div>
+
+- Force mode
+
+Same as velocity mode add the following line to the SDF file.

```

Comment:
Should this instruct the users to replace the plugin they added in the previous step or is this adding a new plugin?